reconcile/utils/saasherder/saasherder.py

```diff
@@ -760,8 +761,8 @@ def _get_file_contents(
     ) -> tuple[Any, str]:
         commit_sha = self._get_commit_sha(url, ref, github)
 
-        if "github" in url:
-            repo_name = url.rstrip("/").replace("https://github.com/", "")
+        if url.startswith(GITHUB_BASE_URL):
+            repo_name = url.removeprefix(GITHUB_BASE_URL).rstrip("/")
```

Comment:
Also here, this pattern is often used.

```
if repo_name := is_github_repo(repo)
```


Maybe a nice utils method that handles both patterns.